icuSources/stubdata/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftFoundation open source project
+##
+## Copyright (c) 2024 Apple Inc. and the SwiftFoundation project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.md for the list of SwiftFoundation project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+target_include_directories(_FoundationICU PRIVATE .)
+target_sources(_FoundationICU
```

Comment:
I don't think we want to do this - there shouldn't be any need to include the stub data in this target since we already have `icu_package_data.cpp` right?

Reply:
Ahh good catch. Thanks!